app/src/main/java/org/mozilla/fenix/ext/ImageButton.kt

```diff
@@ -8,7 +8,7 @@ import android.view.View
 import android.widget.ImageButton
 
 fun ImageButton.hideAndDisable() {
-    this.visibility = View.INVISIBLE
+    this.visibility = View.GONE
```

Comment:
Can we keep this as invisible (or if not make a new fun)? Other places this is used rely on the spacing, ie see the menu hiding in history selection before and after this change:

![Screenshot_20200423-082328](https://user-images.githubusercontent.com/25442310/80117577-33b12200-853c-11ea-87c3-ad4105371e03.png)
![Screenshot_20200423-082532](https://user-images.githubusercontent.com/25442310/80117569-31e75e80-853c-11ea-8fe9-85a963d1e861.png)